src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -79,7 +81,7 @@ public MainViewModel(PowerToysRunSettings settings, CancellationToken nativeThre
             Results = new ResultsViewModel(_settings, this);
             History = new ResultsViewModel(_settings, this);
             _selectedResults = Results;
-
+            Plugins = new ObservableCollection<PluginPair>(PluginManager.AllPlugins.Where(a => !a.Metadata.Disabled && a.Metadata.ActionKeyword != string.Empty));
```

Comment:
Is this always executed when  the window gets opened?

(Asking because of an other PR where I need to update some plugin metadata on launching PT Run.)